Content.Client/BarSign/Ui/BarSignMenu.xaml

```diff
@@ -0,0 +1,16 @@
+<controls:FancyWindow
+    xmlns="https://spacestation14.io"
+    xmlns:controls="clr-namespace:Content.Client.UserInterface.Controls"
+    Title="{Loc 'barsign-ui-menu'}"
+    MinSize="280 180"
+    SetSize="280 180">
+    <BoxContainer VerticalExpand="True" VerticalAlignment="Center" HorizontalAlignment="Center" Margin="10 5 10 10">
+        <BoxContainer Orientation="Vertical" VerticalAlignment="Center" HorizontalAlignment="Center">
+            <Label Text="{Loc 'barsign-ui-set-label'}" HorizontalAlignment="Center"/>
+            <OptionButton Name="SignOptions" HorizontalAlignment="Center" VerticalAlignment="Center" MinSize="175 60"/>
+            <Control MinHeight="20"/>
+        </BoxContainer>
```

Comment:
maybe it would look better if this was orientated horizontally with some padding or a VSeparator and with one of those blueish panels in the background we sometimes use in UIs

Reply:
![image](https://github.com/space-wizards/space-station-14/assets/98561806/3474a1ee-ec99-4976-a8e0-6722455e8e86)

i didnt see the vision but i added some mild flare to make it look less completely placeholder